src/main/java/org/wso2/carbon/esb/connector/auth/AmazonLambdaAuthConnector.java

```diff
@@ -0,0 +1,438 @@
+            if (StringUtils.isNotEmpty(canonicalUri)) {
+                canonicalRequest.append(canonicalUri.replaceAll
+                        (AmazonLambdaConstants.TRIM_SPACE_REGEX, AmazonLambdaConstants.EMPTY_STR))
+                        .append(AmazonLambdaConstants.NEW_LINE);
+            } else {
+                canonicalRequest.append(AmazonLambdaConstants.FORWARD_SLASH)
+                        .append(AmazonLambdaConstants.NEW_LINE);
+            }
+            //Setting canonicalQueryString
+            final StringBuilder canonicalQueryString = new StringBuilder();
+            final Map<String, String> queryParametersMap = ParameterNamesMap.getQueryParameterNamesMap();
+            for (Map.Entry<String, String> entry : queryParametersMap.entrySet()) {
+                String key = entry.getKey();
+                String tempParam = queryParametersValueMap.get(key);
+                if (StringUtils.isNotEmpty((tempParam))) {
+                    queryParamsMap.put(queryParametersMap.get(key), tempParam);
+                }
+            }
+            final SortedSet<String> queryKeys = new TreeSet<>(queryParamsMap.keySet());
+            for (String key : queryKeys) {
+                String queryValue = queryParamsMap.get(key);
+                canonicalQueryString.append(URLEncoder.encode(key, AmazonLambdaConstants.UTF_8))
+                        .append(AmazonLambdaConstants.EQUAL)
+                        .append(URLEncoder.encode(queryValue, AmazonLambdaConstants.UTF_8))
+                        .append(AmazonLambdaConstants.AMPERSAND);
+            }
+            //Removes additional ampersand added at the end from canonicalQueryString and append to canonicalRequest.
+            if (canonicalQueryString.length() > 0) {
+                canonicalRequest.append(canonicalQueryString.substring(0, canonicalQueryString.length() - 1))
+                        .append(AmazonLambdaConstants.NEW_LINE);
+            } else {
+                canonicalRequest.append(AmazonLambdaConstants.NEW_LINE);
+            }
+            //Setting canonicalHeader and signedHeader
+            final Map<String, String> headerParametersMap = ParameterNamesMap.getHeaderParameterNamesMap();
+            final StringBuilder canonicalHeader = new StringBuilder();
+            final StringBuilder signedHeader = new StringBuilder();
+            headersParamsMap.put(AmazonLambdaConstants.API_X_AMZ_DATE, amzDate);
+            for (Map.Entry<String, String> entry : headerParametersMap.entrySet()) {
+                String key = entry.getKey();
+                String tempParam = headerParametersValueMap.get(key);
+                if (StringUtils.isNotEmpty(tempParam)) {
+                    headersParamsMap.put(headerParametersMap.get(key),
+                            tempParam.replaceAll
+                                    (AmazonLambdaConstants.TRIM_SPACE_REGEX, AmazonLambdaConstants.EMPTY_STR));
```

Comment:
Fix formatting issue in above three lines.

Reply:
Fix with 408c7bd